pkgs/development/python-modules/apricot-select/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, scipy
+, tqdm
+, scikit-learn
+, nose
+, numba
+, torch
+, python
+}:
+
+
+buildPythonPackage rec {
+  pname = "apricot-select";
+  version = "0.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-O/hy1D7pavFByeTEDkNZqmyl4wIq5DsqiqRrJJR7i9g=";
+  };
+
+  propagatedBuildInputs = [ nose numpy scipy tqdm numba torch scikit-learn ];
+
+  pythonImportsCheck = [ "apricot" ];
+
+  nativeCheckInputs = [ python ];
+
+  checkPhase = ''
+    ${python.interpreter} setup.py test
+  '';
+
+  meta = with lib; {
+    description = "apricot implements submodular optimization for the purpose of selecting subsets of massive data sets to train machine learning models quickly.";
```

Comment:
Add a changelog pattern.

```
https://github.com/jmschrei/apricot/releases/tag/v${version}

Reply:
There is no changelog for the current version that I can find. The only one available on github is 0.5.0. Pypi also seems to show no changelog.